interface/forms/newpatient/common.php

```diff
@@ -200,22 +200,38 @@ class="css_button link_submit" onClick="top.restoreSession()"><span><?php echo x
      <td class='bold' nowrap><?php echo xlt('Visit Category:'); ?></td>
      <td class='text'>
       <select name='pc_catid' id='pc_catid'>
-	<option value='_blank'>-- <?php echo xlt('Select One'); ?> --</option>
-<?php
- $cres = sqlStatement("SELECT pc_catid, pc_catname, pc_cattype " .
-  "FROM openemr_postcalendar_categories where pc_active = 1 ORDER BY pc_seq ");
- $therapyGroupCategories = array();
- while ($crow = sqlFetchArray($cres)) {
-  $catid = $crow['pc_catid'];
-  if($crow['pc_cattype'] == 3)$therapyGroupCategories[] = $catid;
```

Comment:
i am not seeing this though:
``````
if($crow['pc_cattype'] == 3)$therapyGroupCategories[] = $catid;
``````